client/src/components/Form/LoginForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { useAuth } from "../../hooks/useAuth";
+import { Button, TextField, Typography } from "@material-ui/core";
+import Alert from "@material-ui/lab/Alert";
+
+import Form from "./Form";
+
+export default function LoginForm({ classes }) {
+  const [error, setError] = useState(null);
+  const { register, handleSubmit, errors } = useForm();
+
+  const auth = useAuth();
+
+  const onSubmit = async (data) => {
+    const response = await auth.login(data);
+    if (response) {
+      setError(false);
+      // continue to next page
+      alert("logged in.");
+    } else {
+      setError(true);
```

Comment:
I think we should show a different message depending on if the credentials are invalid or if there was some other network error - can we just return the right message from the back end to display here?